src/main/java/com/mentorship/notificationservice/service/NotificationServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.mentorship.notificationservice.service;
+
+import com.mentorship.notificationservice.domain.Notification;
+import com.mentorship.notificationservice.event.model.OrderEvent;
+import com.mentorship.notificationservice.repository.NotificationRepository;
+import jakarta.transaction.Transactional;
+import java.time.LocalDateTime;
+import lombok.RequiredArgsConstructor;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.stereotype.Service;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class NotificationServiceImpl {
+
+    private final NotificationRepository notificationRepository;
+
+    @RabbitListener(queues = "${spring.rabbitmq.template.default-receive-queue}")
+    public void saveNotificationFromQueue(OrderEvent object) {
```

Comment:
Could u pls explain, why have you name method param 'object'? Maybe let's rename it to event or smt like this?

Reply:
Hm, maybe accidentally